genologics/lims.py

```diff
@@ -585,33 +716,37 @@ def put_batch(self, instances):
                 # Tag is art:details, con:details, etc.
                 example_root = instance.root
                 ns_uri = re.match("{(.*)}.*", example_root.tag).group(1)
-                root = ElementTree.Element("{%s}details" % (ns_uri))
+                root = ElementTree.Element(f"{{{ns_uri}}}details")
```

Comment:
That's a lot of curly braces!

Reply:
Ruff's preference and no obvious way to circumvent it from what I can tell

Reply:
No I did a quick search as well and came to the same conclusion.